src/NHPersistenceCompatibilityTests/PersistenceCompatibilityTests/UpdatingSagaTests.cs

```diff
@@ -125,17 +125,19 @@ public void can_fetch_composite_saga_updated_by_another_version(string sourceVer
             Assert.AreEqual("updated", readData.Composite.Value);
         }
 
-        static string[] NHibernatePackageVersions => new[] { "4.5", "5.0", "6.2", "7.0" };
-
         static object[][] GenerateTestCases()
         {
+            var nuget = new NugetHelper();
+            NHibernatePackageVersions = nuget.GetPossibleVersionsFor("NServiceBus.NHibernate", "4.5.0");
+
             var cases = from va in NHibernatePackageVersions
                         from vb in NHibernatePackageVersions
                         select new object[] { va, vb };
 
             return cases.ToArray();
         }
 
+        private static IEnumerable<string> NHibernatePackageVersions;
```

Comment:
Perhaps I should enable resharper before we look at resharper settings :stuck_out_tongue: